openshift/src/main/java/io/syndesis/openshift/OpenShiftServiceImpl.java

```diff
@@ -16,20 +16,15 @@
 package io.syndesis.openshift;
 
 import java.io.IOException;
-import java.io.StringWriter;
-import java.util.HashMap;
+import java.io.InputStream;
 import java.util.List;
 import java.util.Map;
-import java.util.Properties;
 
 import io.fabric8.kubernetes.api.model.EnvVar;
-import io.fabric8.kubernetes.client.RequestConfig;
 import io.fabric8.openshift.api.model.DeploymentConfig;
 import io.fabric8.openshift.api.model.DeploymentConfigStatus;
 import io.fabric8.openshift.client.NamespacedOpenShiftClient;
-import io.fabric8.openshift.client.OpenShiftClient;
 import io.syndesis.core.Names;
-import io.syndesis.core.SyndesisServerException;
 
 public class OpenShiftServiceImpl implements OpenShiftService {
```

Comment:
nit: would help if all `name` parameters in private methods that receive sanitized name would be named `sName` (or more explicit `sanitizedName`)